libs/login/WordPressLoginFlow/src/main/java/org/wordpress/android/login/LoginEmailFragment.java

```diff
@@ -312,6 +315,12 @@ public void onActivityCreated(@Nullable Bundle savedInstanceState) {
             mIsSocialLogin = savedInstanceState.getBoolean(KEY_IS_SOCIAL);
             mIsDisplayingEmailHints = savedInstanceState.getBoolean(KEY_IS_DISPLAYING_EMAIL_HINTS);
             mHasDismissedEmailHints = savedInstanceState.getBoolean(KEY_HAS_DISMISSED_EMAIL_HINTS);
+            if (savedInstanceState.getBoolean(VALIDITY_EMAIL) || !(savedInstanceState
```

Comment:
Hi again! I don't think this if statement is needed. Similar to `Boolean` values above you can just grab the value and that will take care of it 😄 because if it's `true` or `false` then it will be set and if no value is detected it will be `null`.

So just having this line below works. 
`mIsValidEmail = savedInstanceState.getBoolean(VALIDITY_EMAIL);`